docker-compose.yml

```diff
@@ -1,6 +1,37 @@
-version: "2"
+version: "3.7"
 services:
-  web:
-    image: teamforus/forus-frontend
+  app:
+    image: node:14-alpine
+    container_name: forus-frontend-app
+    command: sh -c "npm install gulp-cli -g && tail -f /dev/null"
+    restart: unless-stopped
+    working_dir: /ng-app/
+    volumes:
+      - ./:/ng-app
+    networks:
+      - forus-frontend
+
+  apache2:
+    image: httpd:2.4
+    container_name: forus-frontend-apache2
+    restart: unless-stopped
+    environment:
```

Comment:
seems unneeded if its already in dockerfile [here](https://github.com/teamforus/forus-frontend/blob/c523e4ee785416146a9785cbaa51923fa9358710/Dockerfile#L24) or is it used somehow?

Reply:
unactual - because dockerfile is not used in docker compose